packages/@aws-cdk/aws-elasticloadbalancingv2/test/alb/load-balancer.test.ts

```diff
@@ -4,8 +4,12 @@ import { Metric } from '@aws-cdk/aws-cloudwatch';
 import * as ec2 from '@aws-cdk/aws-ec2';
 import * as s3 from '@aws-cdk/aws-s3';
 import * as cdk from '@aws-cdk/core';
+import * as cxapi from '@aws-cdk/cx-api';
+import { testFutureBehavior } from 'cdk-build-tools/lib/feature-flag';
 import * as elbv2 from '../../lib';
 
+const s3GrantWriteCtx = { [cxapi.S3_GRANT_WRITE_WITHOUT_ACL]: true };
```

Comment:
I don't recall you using the cxapi import for the other `testFutureBehavior` test updates. Any reason for the switch for this particular flag?

Reply:
You're right. I've been inconsistent.

I've switched all other places that I did in the past to use the format `cxapi.` instead of hardcoding the string.